src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    private static final String VALID_LOGIN = "test_login_created_at " + LocalDateTime.now();

```

Comment:
Does this `time` have a human-readable `toString()`? I think we need to use `DateTimeFormatter` for it to look nice, not sure tho, but please double-check